Cargo.toml

```diff
@@ -59,6 +59,7 @@ tracing = { version = "0.1", features = ["log"] }
 thiserror = "1.0"
 png = "0.17"
 bitflags = "2.4"
+byteorder = "1.5"
```

Comment:
nitpick: Let’s try to not bring too many unrelated changes. We’ll remove this dependency eventually, I don’t think it’s worth to have it in the workspace.

Reply:
This needed a rebase, this is no longer a change in this PR